cmd/workflow-controller/main.go

```diff
@@ -59,6 +59,7 @@ func NewRootCommand() *cobra.Command {
 		podCleanupWorkers       int    // --pod-cleanup-workers
 		cronWorkflowWorkers     int    // --cron-workflow-workers
 		workflowArchiveWorkers  int    // --workflow-archive-workers
+		semaphoreLimitCacheTTL  int    // --semaphore-limit-cache-ttl
```

Comment:
I think this should be in the workflow-controller-configmap instead of as a command line parameter. I'd like this to be changed please.